### PR TITLE
Fix markdown bug

### PR DESCRIPTION
**Pull request by RDIL**
Fixes https://github.com/cirruslabs/cirrus-ci-docs/issues/240

### CODE REVIEW

File: src/components/ViewerBuildList.js [hunk 1]
```diff
@@ -92,7 +91,9 @@ class ViewerBuildList extends React.Component {
         </TableCell>
         <TableCell className={classes.cell}>
           <div className="card-body">
-            <ReactMarkdown className="card-text" source={build.changeMessageTitle}/>
+            <Typography variant="p" color="inherit">
```
Comment (fkorotkov) [inline src/components/ViewerBuildList.js]: I don't think there is such variant. You can probably just use `h6`.

**Comment by fkorotkov** [comment]
Let's then change all the usages of `changeMessageTitle`. 🙌

File: src/components/ViewerBuildList.js (hunk 1 above)
Comment (RDIL) [reply src/components/ViewerBuildList.js]: addressed

**Comment by RDIL** [comment]
will do, need a few minutes though.

**Pull request merged**